nir/src/main/scala/scala/scalanative/nir/serialization/Tags.scala

```diff
@@ -77,18 +77,20 @@ object Tags {
 
   final val Conv = Comp + 32
 
-  final val TruncConv    = 1 + Conv
-  final val ZextConv     = 1 + TruncConv
-  final val SextConv     = 1 + ZextConv
-  final val FptruncConv  = 1 + SextConv
-  final val FpextConv    = 1 + FptruncConv
-  final val FptouiConv   = 1 + FpextConv
-  final val FptosiConv   = 1 + FptouiConv
-  final val UitofpConv   = 1 + FptosiConv
-  final val SitofpConv   = 1 + UitofpConv
-  final val PtrtointConv = 1 + SitofpConv
-  final val InttoptrConv = 1 + PtrtointConv
-  final val BitcastConv  = 1 + InttoptrConv
+  final val SSizeCastConv = 1 + Conv
+  final val ZSizeCastConv = 1 + SSizeCastConv
+  final val TruncConv     = 1 + ZSizeCastConv
```

Comment:
Thoose should be added at the end of Conv block (after BitcastConv), otherwise it would break binary compatibility.

Reply:
The binary compatibility is already going to be quite broken since lots of `Long`s got replaced by `Size`, so thought that since we're breaking compatibility we might as well keep the order of tags tidy.